editor/tests/org.fusesource.ide.camel.editor.tests.integration/META-INF/MANIFEST.MF

```diff
@@ -30,6 +30,7 @@ Require-Bundle: org.junit;bundle-version="4.12.0",
  org.fusesource.ide.camel.model.service.core.tests.integration,
  org.fusesource.ide.branding;bundle-version="10.0.0",
  org.fusesource.ide.projecttemplates;bundle-version="10.0.0",
+ org.fusesource.ide.preferences;bundle-version="10.0.0",
```

Comment:
I think that it is 10.2 (even if we are not taking a great care about it)

Reply:
all the versions should be updated in one commit

Reply:
I saw several manifests pointing to 10.0.0

Reply:
@apodhrad what do you mean exactly  by "it should be updated in one commit"? We should update all of them when upgrading Fuse Tooling version?

As I said previously, we are not taking great care of these requirements version as we always install against the same Target Platform and with All Fuse Tooling plugins of the same version.

I think it is better to use the latest version when adding a new one as we are not checking backward-compatibility.